### ISSUE TITLE
Preview Interface is sortable on browse

### ISSUE THREAD

**Issue by mvhirsch** [directus/app#1990]
Hi folks, I really love Directus!
There is a caveat while sorting via a collections browser list, that should be addressed.
I have a collection, using a "preview"-interface. Even though I've set "hidden on browse", my current version seems to display it anyway. This should be fixed with: https://github.com/directus/app/issues/1720

But for now, I _can_ sort it, but than my whole view is broken, telling me a "Server error has occured". My API backend is logging the following:

```
[2019-08-28 09:30:24] api[_].ERROR: Directus\Database\Exception\UnableSortAliasFieldException: Unable to sort by "product_preview" field because it is an alias. in /var/www/html/src/core/Directus/Database/TableGateway/RelationalTableGateway.php:1573
Stack trace:
#0 [internal function]: Directus\Database\TableGateway\RelationalTableGateway->processSort
#1 /var/www/html/src/core/Directus/Database/TableGateway/RelationalTableGateway.php(799): call_user_func_array
#2 /var/www/html/src/core/Directus/Database/TableGateway/RelationalTableGateway.php(1012): Directus\Database\TableGateway\RelationalTableGateway->applyParamsToTableEntriesSelect
#3 /var/www/html/src/core/Directus/Database/TableGateway/RelationalTableGateway.php(1137): Directus\Database\TableGateway\RelationalTableGateway->fetchItems
#4 /var/www/html/src/core/Directus/Database/TableGateway/RelationalTableGateway.php(819): Directus\Database\TableGateway\RelationalTableGateway->fetchData
#5 [internal function]: Directus\Database\TableGateway\RelationalTableGateway->getItems
```

Since the user was not able to repair this on his own, the only way to fix this was reset the user's collection preset, by `mysql> DELETE FROM directus_collection_presets WHERE id = 27 AND user = 26 LIMIT 1;`

Is there a flag for any Interface to stop allowing "sorting" on items browse list?

----

My versions:

* APP: v7.0.18 (Docker)
* API: v2.0.18 (Docker)
* MySQL: v5.7

**Comment by urvashithakar** [comment]
Hey @mvhirsch,
> But for now, I _can_ sort it, but than my whole view is broken, telling me a "Server error has occured".

This is not only `Docker` specific but occurs in `APP`. Fixed in [#1997](https://github.com/directus/app/pull/1997)

**Comment by mvhirsch** [comment]
Awesome, thank you!
